q2_feature_classifier/_blast.py

```diff
@@ -0,0 +1,77 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2017, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import tempfile
+import pandas as pd
+from os.path import join
+from q2_types.feature_data import (
+    FeatureData, Taxonomy, Sequence, DNAFASTAFormat)
+from qiime2.plugin import Int, Str, Float
+from .plugin_setup import plugin
+from ._consensus_assignment import (_consensus_assignments,
+                                    _parse_params,
+                                    _run_command)
+
+
+def blast(query: DNAFASTAFormat, reference_reads: DNAFASTAFormat,
+          reference_taxonomy: pd.Series, maxaccepts: int=10, min_id: int=0.8,
+          strand: str='both', evalue: float=0.001, t_delim: str=';',
+          min_consensus: float=0.51, unassignable_label: str="Unassigned",
+          num_threads: str=1, params: str=None) -> pd.DataFrame:
+
+    min_id = min_id * 100
+    params = _parse_params(params)
```

Comment:
This makes me a bit nervous. Do you think it's essential? It might represent a security issue (for example, what would happen if a user provided `;,rm,-rf,/`? (Don't try though, because I have no idea what would happen!) It also can be confusing - for example, does providing `evalue` here override the `evalue` parameter?